src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -461,16 +480,42 @@
     }
   }
 
+  .topRowWrapper {
+    display: flex;
+    flex-direction: row;
+    gap: min(var(--space-lg), 50px);
+    position: sticky;
+    top: var(--space-lg);
+    grid-area: top;
+    justify-content: space-between;
+    z-index: 30;
+
+    .opts {
+      display: flex;
+      gap: var(--space-xs);
+      > * {
+        box-shadow: 4px 4px 8px rgba(0, 0, 0, 0.25);
+      }
+    }
+
+    .opts-left {
+      justify-content: flex-start;
+    }
+
+    .opts-right {
+      justify-content: flex-end;
+    }
```

Comment:
these utility classes doesn't seem to be needed, the parent has `justify-content: space-between;` anyway

Reply:
Fixed